cli/src/trusted_base_cli/mod.rs

```diff
@@ -43,9 +43,6 @@ pub enum TrustedBaseCommand {
 	/// send funds from one incognito account to another
 	Transfer(TransferCommand),
 
-	/// ROOT call to set some account balance to an arbitrary number
-	SetBalance(SetBalanceCommand),
```

Comment:
Ok, I see that this is intended. Makes sense to remove this, even for testing we always have access to an account with a lot of funds.